packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -26,7 +26,7 @@ const retryButtonFn = (onRetry: PaymentErrorViewProps['onRetry']) => (
   <Localized id="payment-error-retry-button">
     <button
       data-testid="retry-link"
-      className="button retry-link"
+      className="button retry-link primary-button"
```

Comment:
huh!  So this is a change and not a fix?  It was grey in the past?  I thought it was blue.  🤷

Reply:
It was definitely blue at some point in the past. But CSS is tricksy as if some change to CSS causes this rule to be less specific, it will be ignored in favor of the more specific one for example.